server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryFileCreator.java

```diff
@@ -27,47 +43,55 @@ public void createFiles() throws IOException, JSONException {
         createManifest();
         createBuildpackAdditionsFile();
         createDeployScript();
+        insertFiles();
     }
 
     private void createManifest() throws IOException {
-        String manifestContent;
-        manifestContent = createManifestHead();
-        manifestContent += createEnvironmentVariables();
-        manifestContent += createService();
-        fileAccess.access("/manifest.yml").append(manifestContent).close();
+        createManifestHead();
+        createAttributes();
+        createEnvironmentVariables();
+        createService();
     }
 
-    private String createManifestHead() {
-        return "applications: \n- name: " + app.getAppName();
+    private void createManifestHead() throws IOException {
+        String manifestHead = MANIFESTHEAD + "- " + NAMEBLOCK + ": " + app.getAppName();
+        fileAccess.access("/" + MANIFEST).appendln(manifestHead).close();
     }
 
-    private String createEnvironmentVariables() {
-        String envBlock;
-        envBlock = "\n  env:";
+    private void createEnvironmentVariables() throws IOException {
+        ArrayList<String> environmentVariables = new ArrayList<>();
+        environmentVariables.add("  " + ENVIRONMENT.getName() + ":");
         for (Map.Entry<String, String> entry : app.getEnvironmentVariables().entrySet()) {
-            envBlock = envBlock + "\n    " + entry.getKey() + ": " + entry.getValue();
+            environmentVariables.add("    " + entry.getKey() + ": " + entry.getValue());
+        }
+        for (String env : environmentVariables
+            ) {
+            fileAccess.access("/" + MANIFEST).appendln(env).close();
         }
-        return envBlock;
     }
 
-    private String createService() {
-        String serviceBlock;
-        serviceBlock = "\n  service:";
+    private void createService() throws IOException {
+        ArrayList<String> services = new ArrayList<>();
+        services.add("  " + SERVICE.getName() + ":");
         for (String service : app.getServices()) {
-            serviceBlock += "\n    - " + service;
+            services.add("    - " + service);
+        }
+        
+        for (String service : services
+            ) {
```

Comment:
should be `for (String service: services) {`
Linebreaks like this one violate generally applied java formatting conventions.